icicle/appUtils/ntt/ntt.cu

```diff
@@ -61,7 +58,7 @@ namespace ntt {
       int number_of_threads = MAX_THREADS_BATCH;
       int number_of_blocks = (n * batch_size + number_of_threads - 1) / number_of_threads;
       reverse_order_kernel<<<number_of_blocks, number_of_threads, 0, stream>>>(arr, arr_reversed, n, logn, batch_size);
-      cudaMemcpyAsync(arr, arr_reversed, n * batch_size * sizeof(E), cudaMemcpyDeviceToDevice, stream);
+      cudaMemcpyAsync(arr, arr_reversed, n * batch_size * sizeof(E), cudaMemcpyDefault, stream);
```

Comment:
cudaMemcpyDefault is only on devices that support unified virtual addressing...what devices are these? We should update docs for supported devices

Reply:
@vhnatyk I also don't understand the purpose of this tbh